decentraland/unity-renderer
# feat: load animation on asset reuse

## What does this PR change?

Adds the MaterialTransitionController to all objects on reuse. It applies the change both to AB as well as GLTF.

## How to test the changes?

AB
1. Open https://play.decentraland.zone/?renderer-branch=feat/load-animation-on-reuse
2. Go running to -50,0. See along the way if objects are loaded with a transition effect instead of popping in.

GLTF
1. Open https://play.decentraland.zone/?renderer-branch=feat/load-animation-on-reuse&DISABLE_ASSET_BUNDLES&DISABLE_WEARABLE_ASSET_BUNDLES
2. Go running to -50,0. See along the way if objects are loaded with a transition effect instead of popping in.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
